lib/charms/vault_k8s/v0/vault_kv.py

```diff
@@ -574,6 +590,7 @@ def _handle_relation(self, event: ops.EventBase):
             return
         if self.charm.unit.is_leader():
             relation.data[self.charm.app]["mount_suffix"] = self.mount_suffix
+            relation.data[self.charm.app]["wrap_ttl"] = self.wrap_ttl
```

Comment:
Here we are proposing a change that affects the `vault-kv` charm relation interface, and not only the library that wraps its use. If we need a change to the interface it should be done in the [charm-relation-interfaces](https://github.com/canonical/charm-relation-interfaces/tree/main/interfaces/vault_kv/v0) project first, then let's tackle implementation